src/pallene/coder.lua

```diff
@@ -1821,6 +1842,10 @@ function Coder:generate_luaopen_function()
             #error "Lua version must be exactly 5.4.6"
             #endif
 
+            /* Initialize Pallene Tracer. */
+            /**/
+            pallene_tracer_init(L);
+            /**/
```

Comment:
```suggestion
            pallene_tracer_init(L);
```

Reply:
This is to add newlines before and after `pallene_tracer_init`, so that it looks separated instead of looking blended between.